### PR TITLE
Start simple howto style for page content.

### PR DESCRIPTION
**Pull request by pabzm**
Including two page already adapted.

This is not finished, just a start to show the idea. I'd be happy to work collaboratively on this task, or hand it over. Just comment your thoughts.

-----

Screenshot of page for iCloud:
image

------

Screenshot of page for aktivix:
image

### CODE REVIEW

**Comment by r10s** [comment]
looks great!

the "just works" text is a standard text that is shown for all providers having the "OK" bit set then?

File: _layouts/provider.html [hunk 1]
```diff
@@ -17,54 +17,22 @@
...
-            {%endif%}
-            {%if page.website%}
-                <br>
-                <b>Website</b>: <a href="{{ page.website }}">{{ page.website }}</a>
-            {%endif%}
-        </div>
-        <hr>
-        <div class="section">
-            {{ content }}
+      <img src='/assets/provider-db-logo.svg' class='provider-db-logo' alt=''/>
```
Comment (missytake) [inline _layouts/provider.html]: On the staging instance, the image path seems broken: https://testrun.org/provider/3fd1f6b32ecf43c7834594f7b058744e178a809f/_site/aktivix-org

`'./assets[...]` seems to do the trick :)

**Comment by pabzm** [comment]
> the "just works" text is a standard text that is shown for all providers having the "OK" bit set then?

Yes!

File: _layouts/provider.html (hunk 1 above)
Comment (pabzm) [reply _layouts/provider.html]: Good catch, thanks! Fixed in 8ed07e8.

**Comment by Simon-Laux** [comment]
I like the emoji

**Comment by pabzm** [comment]
I pushed some more provider pages converted to the new data structure.

Please note I had to change one field: "date" as top-level field requires a valid date format. I found that too limiting, thus I changed the field name to "last_checked", which I found more telling anyway.

Also "%" has a special meaning in YAML and apparently it's not enough to quote that character (in "%EMAILLOCALPART%", but it must be pushed into a multi-line-string. That's unfortunate I think, we should maybe change our "encoding".

**Comment by r10s** [comment]
`last_checked` is okay for me.

wrt username: maybe we can use sth. as `username_pattern=email|emaillocalpart` where `email` is the default, tbh i cannot imaging that his can ever be something literal. and if, can still use `username=...`

**Comment by Simon-Laux** [comment]
I agree, if we should ever come accross anything that is not in such enum we can implement a username field. Generaly we can solve this problem when it occurs.

**Comment by pabzm** [comment]
👍 for `username_pattern`.

I'd only suggest to use the strings from Thunderbirds autoconfig DB stripped by the '%'. Thus: `emailaddress|emaillocalpart`.

**Comment by r10s** [comment]
so, not sure about the WIP, but i think, we should merge that and iterate from there in smaller prs.

**Comment by pabzm** [comment]
Merged to make way for follow-up PRs.

**Pull request merged**